src/dotnet/commands/dotnet-compile/Program.cs

```diff
@@ -71,16 +71,35 @@ private static bool OnExecute(List<ProjectContext> contexts, CompilerCommandApp
 
             var managedOutput = outputPaths.CompilationFiles.Assembly;
 
+            // Create the library exporter
+            var exporter = context.CreateExporter(args.ConfigValue);
```

Comment:
Yes. We need to in the long term. But we wanted to have simple cases work without blocking on the tons of work. We got rid of System.Console from AppDep and it now comes from the project file. System.IO.FileSystem comes from the project file. When we have the AOT rid, we'll look for it -- tracked by #402. By design and what you say is future work. The AOT libraries will override just like AppDep overrides in the PR.

https://github.com/dotnet/cli/issues/402